dusty/systems/virtualbox/__init__.py

```diff
@@ -140,6 +140,22 @@ def _apply_nic_fix():
     log_to_client('Setting NIC 1 to use PCnet-FAST III')
     check_call_demoted(['VBoxManage', 'modifyvm', constants.VM_MACHINE_NAME, '--nictype1', constants.VM_NIC_TYPE])
 
+def delete_docker_vm_host_only_interface():
+    """Attempt to delete the host-only interface attached
+    to the current Dusty VM. VM should be stopped
+    before calling this."""
+    adapter_name = get_vm_hostonly_adapter()
+    log_to_client('Deleting host-only interface {}'.format(adapter_name))
+    check_call_demoted(['VBoxManage', 'hostonlyif', 'remove', adapter_name])
+
```

Comment:
yeah the other one is the forwarding interface that lets you talk to the internet